logstash-core-event-java/src/main/java/com/logstash/Javafier.java

```diff
@@ -1,42 +1,49 @@
 package com.logstash;
 
-import org.jruby.RubyArray;
-import org.jruby.RubyHash;
-import org.jruby.RubyString;
-import org.jruby.RubyObject;
-import org.jruby.RubyBoolean;
-import org.jruby.RubyArray;
-import org.jruby.RubyFloat;
-import org.jruby.RubyInteger;
-import org.jruby.RubyNil;
-import org.jruby.RubyBoolean;
-import org.jruby.RubyFixnum;
-import org.jruby.RubyTime;
-import org.jruby.RubySymbol;
-import org.jruby.RubyBignum;
-import org.jruby.ext.bigdecimal.RubyBigDecimal;
 import com.logstash.ext.JrubyTimestampExtLibrary;
-import org.jruby.runtime.builtin.IRubyObject;
-import java.math.BigDecimal;
 import org.joda.time.DateTime;
+import org.jruby.*;
```

Comment:
Not sure where this was discussed but I would prefer we keep explicit classes imports?